daemon/tests/harness/mocks/oracle.rs

```diff
@@ -53,3 +53,7 @@ pub trait Oracle {
 pub fn dummy_announcement() -> oracle::Announcement {
     OliviaData::example_0().announcement()
 }
+
+pub fn dummy_end_attestation() -> oracle::Attestation {
```

Comment:
I think we should inline these. It would be good to see in the test that we are using `example_0` for both.

Reply:
inlined! and used a common struct in the test to see that they're matched from the same source (`oracle_data`)